### PR TITLE
[style] Add custom cluster icons and adjust zoom on cluster clicks

### PR DESCRIPTION
**Pull request by ethan-tam33**
### Description
[//]: # "[Required] Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

- used `.addListener` within `clusterer` object in `AddMarkers.tsx` to set the zoom value if the number of markers in the cluster are 2 (the only cases where the markers go beyond the screen when the cluster is clicked)
- minimized zooming effect on cluster click
- added the SVG of the custom cluster icons from Josh's Figma (but had to add a few lines since Figma hardcoded the "33" value for some reason)
- created a `renderer` object that adds the custom icon for each cluster
- changed marker and cluster icon size to be smaller

### Screenshots
[//]: # "Add any screenshots that will help your PR reviewer understand what’s going on in the PR!"
[//]: # "[Required] For frontend tasks: add screenshots of the layout and expected behavior (GIFs if you’re feeling extra fancy)!"
[//]: # "Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 
Screenshot 2024-11-16 at 4 55 15 PM

Screenshot 2024-11-17 at 7 17 27 PM

## How to review
[//]: # "/[Required] Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"

- review `AddMarkers.tsx`

## Next steps
[//]: # "[Optional] - What has not yet been added to the PR, doesn’t yet have any functionality — essentially, what needs to be fixed or added."
[//]: # "[Optional] any temporary fixes in this PR that should be cleaned up later."

- better understand if it's possible to eliminate zooming animation on cluster click

### Online sources
[//]: # "[Optional] insert links to any useful tutorials or documentation that you used for this PR!"

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @itsliterallymonique

### CODE REVIEW

File: assets/Clusters/icons.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
+import { CoinbaseText } from '@/styles/fonts';
+
+export const ClusterIcon = ({ count }: { count: number }) => (
+  <svg
+    width="64"
```
Comment (itsliterallymonique) [inline assets/Clusters/icons.tsx:5]: can you change these width and height from image to rem? I believe you can write it as like "3rem" and it should work.

Comment (itsliterallymonique) [reply assets/Clusters/icons.tsx:5]: but if it's not possible, that's fine -- go ahead and squash and merge!

**Review by itsliterallymonique — APPROVED** [review]
If you can change the SVG clustering width from pixes to rem (check my in-code comment) that would be amazing. If not -- go ahead and squash and merge!
